the-birthday-times.js

```diff
@@ -3,32 +3,28 @@
 const needle = require('needle')
 const dayjs = require('dayjs')
 const prompts = require('prompts')
+const dedent = require('dedent')
 const color = require('ansi-colors')
+const { setTimeout } = require('timers/promises')
 
-function main () {
-  try {
-    if (!process.env.NYTIMES_KEY) {
-      throw new Error(color.red("Please set your API key on your operating system.\nTo set environment variables on macOS or Linux, run the export command from the terminal: export NYTIMES_KEY='YOUR-API-KEY'\n"))
-    }
-    showFirstMessage()
-    getBirthdayNews()
-  } catch (error) {
-    console.log(error.message)
-  }
+async function main () {
+  showTitle()
+  const birthday = new Birthday(await getBirthday())
+  await loadingMessage()
+  birthday.displayNews(await birthday.getNews())
 }
 
-function showFirstMessage () {
-  console.log('\n' +
-':￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣: \n' +
-':                                                                      :')
-  console.log(color.bold(':              🗞   T h e   B i r t h d a y   T i m e s 🗞               :'))
-  console.log(
-    ':                                                                      : \n' +
-':                                                                      : \n' +
-' ￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣ \n\n')
+function showTitle () {
+  const title = dedent`:￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣:
+    :                                                                      :
+    :              ${color.bold('🗞   T h e   B i r t h d a y   T i m e s   🗞')}             :
+    :                                                                      :
+    :                                                                      :
+     ￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣`
```

Comment:
dedent を使っているのであれば以下のようにしたらいいんじゃなかなと思いました。

```suggestion
  const title = dedent`
    :￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣:
    :                                                                      :
    :              ${color.bold('🗞   T h e   B i r t h d a y   T i m e s   🗞')}             :
    :                                                                      :
    :                                                                      :
     ￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣￣
  `
```